azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachines.java

```diff
@@ -32,20 +32,73 @@
     PagedList<VirtualMachineSize> listSizes(String region) throws CloudException, IOException;
```

Comment:
Today found that there are two types of listing of VirtualMachineImages. 
1. List the available size in a region
2. List the available sizes a VM can be resized to

For 1 i gave the name listSizes and for 2 it's named as availableSizes.

Any suggestion for better name in this case?